### PR TITLE
amourgus + xtras

### PR DESCRIPTION
**Pull request by ghost**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: amogus
add: amojoust in det office and maint
add: amojus as heirloom item for jobs not assigned heirloom 
add: amogert traitor item
fix: fixed typo in syndie surgery bag uplink item
add: robotnik mustache facial hair option
add: exclamation mohawk hair option
tweak: unce machine begins off roundstart (DAMN THAT IS ANNOYING!!)
add: 4 new fart messages
/:cl:

## About The Pull Request
tenor (3)

## Why It's Good For The Game
amongos
tenor (4)

### CODE REVIEW

**Comment by carlopault** [comment]
+1 all of this stuff looks really neat. @yoyobatty merge dis

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -353,7 +360,7 @@
 		if(mood_message in vowbroken_message)
 			mood_message = null
 	cheer_up()
-
+// gay ^^^
```
Comment (yoyobatty) [inline code/game/objects/items/plushes.dm:363]: no

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -494,6 +501,109 @@
...
+		user.visible_message("<span class='suicide'>The plushie votes [user]'s head off their shoulders!</span>")
+		return (BRUTELOSS)
+
+/obj/item/toy/plush/amongus/imposter/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] tells the plushie they can't take it anymore! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	playsound(src, 'sound/effects/amogussplurch.ogg', 100, 1, 5)
+	playsound(src, 'sound/effects/killsfx.ogg', 100, 1, 5)
+	H.gib()
+	return
```
Comment (yoyobatty) [inline code/game/objects/items/plushes.dm]: should be `return BRUTELOSS`

Comment (ghost) [reply code/game/objects/items/plushes.dm]: fixed

File: code/game/objects/items/plushes.dm [hunk 3]
```diff
@@ -494,6 +501,109 @@
...
+	attack_verb = list("susses", "punches", "slaps")
+	squeak_override = list('sound/effects/amogus.ogg', 'sound/effects/amoguspanch.ogg', 'sound/effects/amogussplurch.ogg', 'sound/effects/bong.ogg' = 1)
+
+/obj/item/toy/plush/amongus/Initialize() // different color amojus
+	. = ..()
+	var/pickcolor = pick("red_crewmate","cyan_crewmate","gray_crewmate","blue_crewmate","green_crewmate","orange_crewmate","gay_crewmate","yellow_crewmate","really_gay_crewmate")
+	icon_state = "[pickcolor]"
+	item_state = "red_crewmate"
+
+/obj/item/toy/plush/amongus/Initialize() //rare amojus - 2do give greentext if u escape with it
```
Comment (yoyobatty) [inline code/game/objects/items/plushes.dm]: initialized twice, fix this

**Review by yoyobatty — CHANGES_REQUESTED** [review]

File: code/game/objects/items/plushes.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/plushes.dm:363]: Yea

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**